File: src/ThreadAddQty.java
public class ThreadAddQty extends Thread {
    Stocks s;
    int timing;
    int qty;

    public ThreadAddQty(Stocks s, int timing, int qty) {
        this.s = s;
        this.timing = timing;
        this.qty = qty;
    }

    public void run(){
        
        try {
            s.addQty(qty);
            sleep(timing);
            
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
    }
    
}

File: pythonVersion/src/ThreadAddQty.py
from Stocks import Stocks
import threading
import time



class ThreadAddQty(threading.Thread):    
    
    def __init__(self, s:Stocks, qty:int, t:int):
        self.s = s
        self.qty = qty
        self.t = t
        
        threading.Thread.__init__(self)
        
    def run(self):
        self.s.addQty(self.qty)
        time.sleep(self.t)